content/css/concepts/background/terms/background-origin/background-origin.md

```diff
@@ -37,35 +37,47 @@ Set background positioning to the edge of the content box:
 
 ```css
 .main {
-  border: 5px dashed #f1f1f1;
-  padding: 15px;
-  background: linear-gradient(purple, yellow) no-repeat;
+  border: 5px dashed #4d4d4d;
+  padding: 1rem;
+  background: url(./media/background-origin.jpg) no-repeat;
   background-origin: content-box;
 }
 ```
+The resulting output of the code block above is
+
+![Background origin set to padding-box](./media/background-origin-content.png "background-origin: padding-box")
+
 
 ## Example 2
 
-Set background positioning to the inside edge of the border:
+Set background positioning to the inside edge of the border, respecting its' padding:
 
 ```css
 .main {
-  border: 5px dashed #f1f1f1;
-  padding: 15px;
-  background: linear-gradient(purple, yellow) no-repeat;
+  border: 5px dashed #4d4d4d;
+  padding: 1rem;
+  background: url(./media/background-origin.jpg) no-repeat;
   background-origin: padding-box;
 }
 ```
+The resulting output of the code block above is
+
+![Background origin set to padding-box](./media/background-origin-padding.png "background-origin: padding-box")
+
 
 ## Example 3
 
 Set background positioning to the edge of the border:
 
 ```css
 .main {
-  border: 5px dashed #f1f1f1;
-  padding: 15px;
-  background: linear-gradient(purple, yellow) no-repeat;
+  border: 5px dashed #4d4d4d;
+  padding: 1rem;
+  background: url(./media/background-origin.jpg) no-repeat;
   background-origin: border-box;
 }
 ```
+The resulting output of the code block above is
+
+![Background origin set to padding-box](./media/background-origin-border.png "background-origin: padding-box")
```

Comment:
You need to format links as specified in the style guide [here](https://github.com/SSwiniarski/docs/blob/main/documentation/style-guide.md#file-hosting) or else the images will not work when this is published to the website.